src/components/registration/registration.tsx

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import { StyleGenerator } from '../styling/style-generator.service';
+import { OffsetStyle } from '../styling/offset-style';
+import { htmlAttributes } from 'sitefinity-react-framework/widgets/attributes';
+import { classNames } from 'sitefinity-react-framework/utils/classNames';
+import { WidgetContext } from 'sitefinity-react-framework/widgets/widget-context';
+import { PostRegistrationAction } from './interfaces/PostRegistrationAction';
+import { StylingConfig } from '../styling/styling-config';
+import { getUniqueId } from 'sitefinity-react-framework/utils/getUniqueId';
+import { RestExtensionsService } from 'sitefinity-react-framework/sdk/rest-extensions';
+import { RestSdkTypes, RestService } from 'sitefinity-react-framework/sdk/rest-service';
+import { MixedContentContext } from 'sitefinity-react-framework/widgets/entities/mixed-content-context';
+import { ExternalLoginBase } from 'sitefinity-react-framework/login/external-login-base';
+import { ExternalProvider } from 'sitefinity-react-framework/sdk/dto/external-provider';
+import { RegistrationSettingsDto } from 'sitefinity-react-framework/sdk/dto/registration-settings';
+import { RegistrationForm } from './registration-form';
+
+const defaultMixedContent = {
+    ItemIdsOrdered:null,
+    Content:[ {
+        Type:RestSdkTypes.Pages,
+        Variations:null
+    }]
+};
+const EncryptedParam = 'qs';
+
+const isAccountActivationRequest = (context: any) => {
+    if (context.IsLive) {

```

Comment:
[Suggestion]:
``` 
if (context && context.IsLive && context.searchParams[EncryptedParam]) {
    return true;
 }

return false;
```

Reply:
cool! - my brain couldn't compile this one from C# :)